### ISSUE TITLE
Ch. 2.3: location of lake build outputs moved in 4.3.0

### ISSUE THREAD

**Issue by tmoux** [leanprover/fp-lean#141]
Please quote the text that is incorrect:

> To build the package, run the command `lake build`. After a number of build commands scroll by, the resulting binary has been placed in `build/bin`. Running `./build/bin/greeting` results in Hello, world!.

In what way is this incorrect?

Since [4.3.0](https://github.com/leanprover/lean4/releases/tag/v4.3.0), `lake build`  by default puts all build outputs in the `.lake/` directory, so the path to the executable should be `.lake/build/bin/greeting`. This can be confusing to people who run `lake build` and don't immediately see where the build folder is.

**Comment by david-christiansen** [comment]
You're right - the intro to the book states which version it follows, and it could use a bump soon, but not everyone will read that bit of the introduction.

Luckily for me, my past self set it up so these things mostly get tested during the build of the book text, but I'll double-check that this gets fixed on my next round of updates.

Thanks!

**Comment by david-christiansen** [comment]
This is updated in the latest release. Thanks!
